doc/contributing.md

```diff
@@ -78,17 +80,17 @@ Other Helpful information includes:
 
 ### To Spec or not to Spec
 
-Some issues/features may be quick and simple to describe and agree upon. For such scenarios, once a team member has agreed your approach, skip ahead to "Fork, Branch, and Create your PR". Small issues that do not require a spec will be labelled Issue-Bug.
+Some issues/features may be quick and simple to describe and agree upon. For such scenarios, once a team member has agreed your approach, skip ahead to "Fork, Branch, and Create your PR". Small issues that do not require a spec will be labelled Issue-Bug or Issue-Task.

```

Comment:
labeled or labelled?

I was taught single L, but I think both are correct. As long as we're consistent. (It was changed to single L below but not here.)